ratis-common/src/main/java/org/apache/ratis/protocol/Message.java

```diff
@@ -65,6 +68,10 @@ static int getSize(Message message) {
   ByteString getContent();
 
   default int size() {
-    return Optional.ofNullable(getContent()).map(ByteString::size).orElse(0);
+    int size = Optional.ofNullable(getContent()).map(ByteString::size).orElse(0);
+    if (size > 0 && size < (double) (ONE_MB.getSizeInt())) {
+      return 1;
+    }
+    return size/ SizeInBytes.valueOf("1mb").getSizeInt();
```

Comment:
Since this method is a in the protocol package.  We cannot change it.  Otherwise, it is an incompatible change.

Reply:
Reverted this change.